src/Context/SoapContext.php

```diff
@@ -273,4 +273,18 @@ public function beforeStepCheckForException(BeforeStepScope $scope)
             throw $this->fault;
         }
     }
+
+    /**
+     * @Given I register the following SOAP headers:
+     */
+    public function iRegisterHeaders(TableNode $headers)
+    {
+        $soapHeaders = array();
+
+        foreach ($headers->getColumnsHash() as $header) {
```

Comment:
Yes, I guess, and we can make the namespace optional later, but for this issue it should be enough, we'll focus on refinements later. I would like to document usage of this step, but we don't have docs yet, so I will do that later.

Thanks!